prometheus-exporters/ucs-exporter/alerts/ucsmcrcerrors.alerts

```diff
@@ -2,13 +2,15 @@ groups:
 - name: ucsmcrcerrors.alerts
   rules:
   - alert: UcsmCRCError
-    expr: max_over_time(ucsm_crc_error[20m]) > 0
+    expr: increase(ucsm_crc_error[30m]) > 100
```

Comment:
`increase` should only be used with counter metrics (check: https://prometheus.io/docs/prometheus/latest/querying/functions/#increase)

The needed function would be `delta`

Reply:
If you change the function to `delta`, we can close this PR: https://github.com/sapcc/helm-charts/pull/2306